containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx

```diff
@@ -0,0 +1,82 @@
+
+One of the standout benefits of routed IPs is their support for [IP mobility](https://www.scaleway.com/en/docs/compute/instances/concepts/#ip-mobility), streamlining IP management and movement within the Scaleway ecosystem. By simplifying the process of reallocating IPs across different products, routed IPs empower users with unprecedented flexibility and control over their network infrastructure.
+
+Additionally, routed IPs unlock the ability to attach multiple IPv4 addresses to a single Instance, enabling specific functionalities such as IP failover. While this feature may have limited applicability within Kubernetes environments, it offers invaluable flexibility and resilience across other Scaleway products.
+
+<Message type="note">
+  Routed IPs provide primary support for IPv6, ensuring direct connectivity for IPv6 addresses with Instances.
+  However, IPv6 support for Kapsule is not yet available due to compatibility limitations with existing Container Network Interfaces (CNIs).
+</Message>
+
+<Message type="important">
+    NAT IPs will reach end-of-life in Q4/2024. After this date, Scaleway will organize migrations for all clusters to transition to routed IPs.
+    Migrations will be planned region by region, with specific dates to be announced.
+</Message>
+
+## Moving your Kubernetes nodes to routed IPs
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- Set up a [Kubernetes Kapsule cluster](/containers/kubernetes/how-to/create-cluster)
+
+<Message type="note">
+  Newly created Kapsule clusters are deployed with IP mobility enabled by default. For existing clusters, you can launch the migration to routed IPs manually.
+</Message>
+
+### Using the Scaleway console
+
+1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com) side menu. The Kubernetes Kapsule overview displays.
+2. Click the name of the cluster you want to access. The cluster overview page displays.
+3. Click the **Network** tab. The cluster network information displays.
+4. Click **Move to routed IP** in the **IP address type** section of the page. A pop-up displays.
+5. Confirm the move of your worker nodes to routed IPs by typing **MOVE**, then click **Move to routed IPs** to launch the process.
+    <Message type="important">
+      * This operation is similar to a cluster upgrade. Each node in your cluster will reboot. Ensure having more than one node ready in order to avoid downtime.
```

Comment:
```suggestion
      * This operation is similar to a cluster upgrade. Each node in your cluster will reboot. Ensure that you have more than one node ready in order to avoid downtime.
```